test/passing/tests/rec_str_item_comment.ml

```diff
@@ -0,0 +1,10 @@
+module rec A : A =
```

Comment:
This would fit nicely in the existing test `comment_in_modules.ml` instead of a new one.